File: python/program/TV.py
from produk_audio_visual import ProdukAudioVisual

class TV(ProdukAudioVisual):
    def __init__(self, id_produk, nama_produk, merk, harga, garansi, tipe, ukuran, resolusi, jenis_panel):
        super().__init__(id_produk, nama_produk, merk, harga, garansi, tipe)
        self.__ukuran = ukuran
        self.__resolusi = resolusi
        self.__jenis_panel = jenis_panel

    # Getter
    def get_ukuran(self):
        return self.__ukuran

    def get_resolusi(self):
        return self.__resolusi

    def get_jenis_panel(self):
        return self.__jenis_panel

    # Setter
    def set_ukuran(self, ukuran):
        self.__ukuran = ukuran

    def set_resolusi(self, resolusi):
        self.__resolusi = resolusi

    def set_jenis_panel(self, jenis_panel):
        self.__jenis_panel = jenis_panel

File: php/program/TV.php
<?php

require_once 'ProdukAudioVisual.php';

class TV extends ProdukAudioVisual {
    private $ukuran;
    private $resolusi;
    private $jenisPanel;

    public function __construct($idProduk, $namaProduk, $merk, $gambar, $harga, $garansi, $tipe, $ukuran, $resolusi, $jenisPanel) {
        parent::__construct($idProduk, $namaProduk, $merk, $gambar, $harga, $garansi, $tipe); // Meneruskan parameter gambar
        $this->ukuran = $ukuran;
        $this->resolusi = $resolusi;
        $this->jenisPanel = $jenisPanel;
    }

    // Getters
    public function getUkuran() { return $this->ukuran; }
    public function getResolusi() { return $this->resolusi; }
    public function getJenisPanel() { return $this->jenisPanel; }

    // Setters
    public function setUkuran($ukuran) { $this->ukuran = $ukuran; }
    public function setResolusi($resolusi) { $this->resolusi = $resolusi; }
    public function setJenisPanel($jenisPanel) { $this->jenisPanel = $jenisPanel; }
}